the-blue-alliance-ios/View Controllers/MyTBA/MyTBAViewController.swift

```diff
@@ -148,6 +149,11 @@ class MyTBAViewController: ContainerViewController, GIDSignInUIDelegate {
     
     // MARK: - Interface Methods
     
+    @IBAction func settingsTapped(_ sender: UIBarButtonItem) {
```

Comment:
Can we make Settings it's own tab? It's weird to have it live on the myTBA view controller. I know that's how Android does it, but Android is wrong 😛